pkgs/games/steam/fhsenv.nix

```diff
@@ -222,6 +225,8 @@ in buildFHSUserEnv rec {
         export TZ="$new_TZ"
       fi
     fi
+
+    export XDG_DATA_DIRS=${gsettings-desktop-schemas}/share/gsettings-schemas/${gsettings-desktop-schemas.name}:${gtk3}/share/gsettings-schemas/${gtk3.name}:$XDG_DATA_DIRS
```

Comment:
Not needed since https://github.com/NixOS/nixpkgs/pull/161739

Reply:
It doesn't work without it. Any idea what could cause that?